### PR TITLE
Jade compat

### PR DESCRIPTION
**Pull request by IchHabeHunger54**
This PR adds compatibility with Jade, similar to the compatibility with The One Probe already in the mod.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -174,6 +174,13 @@ repositories {
         name = 'ModMaven'
         url = 'https://modmaven.k-4u.nl'
     }
+    maven {
+        name = 'CurseMaven'
+        url = 'https://www.cursemaven.com'
+        content {
+            includeGroup 'curse.maven'
+        }
+    }
```
Comment (Minecraftschurli) [inline build.gradle]: Is jade available anywhere else? I really don't want to have to use cursemaven

File: build.gradle [hunk 2]
```diff
@@ -200,6 +207,8 @@ dependencies {
     modAPI fg.deobf("mcjty.theoneprobe:theoneprobe:${project.top_version}:api") { transitive = false }
     runtimeMod fg.deobf("mcjty.theoneprobe:theoneprobe:${project.top_version}") { transitive = false }
 
+    implementation fg.deobf("curse.maven:jade-324717:${project.jade_version}")
+
```
Comment (Minecraftschurli) [inline build.gradle]: look at the other dependencies we don't use implementation directly

**Review by Minecraftschurli — CHANGES_REQUESTED** [review]
2 things and you are missing the mods.toml entries for this and the other just merged compat

File: build.gradle (hunk 1 above)
Comment (IchHabeHunger54) [reply build.gradle]: If it is, their documentation doesn't mention it. It specifically tells you to use CurseMaven here: https://jademc.readthedocs.io/en/latest/plugins19/getting-started/

Comment (Minecraftschurli) [reply build.gradle]: please use the modrinth maven instead

**Review by Minecraftschurli — APPROVED** [review]

**Pull request merged**